pom.xml

```diff
@@ -653,7 +657,7 @@
         </profile>
         <profile>
         <!-- SUP-420 case. Tomcat 9.0.z and 5.3.z hazelcast. Using jdk 1.8 because tomcat supports 1.8.
-         Therefore we have to exclude jetty tests from compiling/running and jetty classes from classpath
+         Therefore, we have to exclude jetty tests from compiling/running and jetty classes from classpath
           because they require jdk 11. -->
             <id>tomcat-jdk8</id>
```

Comment:
this may not be working anymore, I think we should change a lot of dependency versions in this profile. we can even remove this profile, because tomcat 9 is not supported by this version of hazelcast-wm

Reply:
We can remove it later. This module needs more work. Let's first upgrade to jakarta. Then we can remove these kind of things later